contracts/exchange-wrappers/Starkware.sol

```diff
@@ -15,12 +15,38 @@ pragma solidity ^0.8.0;
 
 
 import { I_StarkwareContract } from "../external/I_StarkwareContracts.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "hardhat/console.sol";
 
-contract Starkware is I_StarkwareContract {
+contract MockStarkware is I_StarkwareContract {
+
+    // ============ State Variables ============
+
+    address immutable USDC_ADDRESS;
```

Comment:
I'm worried I misled you earlier about saying variables that are `immutable` can not have an interface type. It seems like we did that in the claims proxy...

```
contract ClaimsProxy {
  ...
  // ============ Constants ============

  ISafetyModuleV1 public immutable SAFETY_MODULE;
  ILiquidityStakingV1 public immutable LIQUIDITY_STAKING;
  IMerkleDistributorV1 public immutable MERKLE_DISTRIBUTOR;
  ITreasuryVester public immutable REWARDS_TREASURY_VESTER;

  ...
}
```

Mind trying out making `USDC_ADDRESS` of type `IERC20` again now that you're on `v0.8.0`?